awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/AwsAppSignalsCustomizerProvider.java

```diff
@@ -48,8 +48,8 @@
  *   <li>Add AwsMetricAttributesSpanExporter to add more attributes to all spans.
  * </ul>
  *
- * <p>You can control when these customizations are applied using the property otel.smp.enabled or
- * the environment variable OTEL_SMP_ENABLED. This flag is enabled by default.
+ * <p>You can control when these customizations are applied using the property otel.aws.app.signals.enabled or
+ * the environment variable OTEL_AWS_APP_SIGNALS_ENABLED. This flag is disabled by default.
```

Comment:
Oh man I thought we fixed the "disabled by default" comment a long time ago... Good catch.